web/src/lib/clients/ecomap/http.ts

```diff
@@ -1,8 +1,9 @@
-import { navigate } from "svelte-routing";
 import createClient, { type Middleware } from "openapi-fetch";
-import type { components, paths } from "../../../../api/ecomap/http";
-import { clearToken, getToken } from "../../utils/auth";
-import { CommonRoutes } from "../../../routes/constants/routes";
+
```

Comment:
Unnecessary line?

Reply:
Actually it's meant to be here. New lines are used to separate import groups. Since `"openapi-fetch"` is an external package, it needs to be separated from the project's alias variables, e.g. `$api`.   

You can see the order of each group [here](https://github.com/Goncalo-Marques/ecomap/pull/265/files/4c0d5dcc832da55d3637cc953b4e0957f4685425#diff-43a46636688b713076d1995b828f48255b4b0f86efc1edc1d6375307c1e73037R45-R60).

Reply:
ah ok, makes sense